Add Profiling related documentations (#10612)
apache/skywalking: docs/en/setup/backend/backend-ebpf-profiling.md

3. **Start Time**: Whether the current task needs to be executed immediately or at a future point in time.
4. **Duration**: The execution time of the current profiling task.

Once the task is created, the eBPF agent would periodically request from the OAP whether there are any eligible tasks among all the processes collected by the current eBPF agent. 
When the eBPF agent receives a task, it would start the profiling task with the process.

#### Profiling analyze

Once the eBPF agent starts a profiling task for a specific process, it would periodically collect data and report it to the OAP. 
At this point, a scheduling of task is generated. The scheduling data contains the following information:

1. **Schedule ID**: The ID of current schedule.
2. **Task**: The task to which the current scheduling data belongs.
3. **Process**: The process for which the current scheduling Profiling data is being collected.
4. **Start Time**: The execution start time of the current schedule.
5. **End Time**: The time when the last sampling of the current schedule was completed.

> At this point, we can use the existing scheduling ID and time range to query the CPU execution situation of the specified process within a specific time period. 

Reviewer: ```suggestion
Once the schedule is created, we can use the existing scheduling ID and time range to query the CPU execution situation of the specified process within a specific time period. 
```
